packages/devtools_app/lib/src/ui/utils.dart

```diff
@@ -178,6 +182,27 @@ class _OffsetScrollbarState extends State<OffsetScrollbar> {
   }
 }
 
+void scrollToPosition(
+  ScrollController scrollController,
+  double position,
+) {
+  final extentVisible = Range(
+    scrollController.offset,
+    scrollController.offset + scrollController.position.extentInside,
+  );
+
+  if (!extentVisible.contains(position)) {
+    final positionToScrollTo = max(0.0, position - defaultRowHeight);
+
+    scrollController.animateTo(
+      //TODO: should be positionToScrollTo.clamp(0.0, scrollController.position.maxScrollExtent) but maxScrollExtent is not being updated
```

Comment:
Open a bug and change TODO format to:

// TODO(your name): Description, link to issue